evi check/Form1.cs

```diff
@@ -0,0 +1,650 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace evi_check
+{
+    public partial class Form1 : Form
+    {
+        string evi1 = null;
+        string evi2 = null;
+        string evi3 = null;
+        Dictionary<string, List<string>> ghostlist = new Dictionary<string, List<string>>();
+        
+
+
+
+
+        public Form1()
+        {
+            InitializeComponent();
+            monsterlist();
+            Show_output();
+            
+        }
+
+        public Dictionary<string, List<string>> monsterlist()
+            {
+
+          //  Dictionary<string, Dictionary<string,string>> biglist = new Dictionary<string, Dictionary<string, string>>();
+            Dictionary<string, List<string>> smalllist = new Dictionary<string, List<string>>();
+
+            List<string> Banshee = new List<string> {"EMF5","FingerPrints","Cold" };
+            List<string> Demon = new List<string> {"GhostWriting","SpiritBox","Cold" };
+            List<string> Jinn = new List<string> {"EMF5","SpiritBox","GhostOrb" };
+            List<string> Mare = new List<string> {"SpiritBox","Cold","GhostOrb" };
+            List<string> Onie = new List<string> {"EMF5","GhostWriting","SpiritBox" };
+            List<string> Phantom = new List<string> {"EMF5","Cold","GhostOrb" };
+            List<string> Poltergeist = new List<string> {"FingerPrints","SpiritBox","GhostOrb" };
+            List<string> Revenant = new List<string> {"EMF5","GhostWriting","FingerPrints" };
+            List<string> Shade = new List<string> {"EMF5","GhostWriting","GhostOrb" };
+            List<string> Spirit = new List<string> {"GhostWriting","FingerPrints","SpiritBox" };
+            List<string> Wraith = new List<string> {"FingerPrints","SpiritBox","Cold"};
+            List<string> Yurei = new List<string> {"GhostWriting","Cold","GhostOrb" };
+
+
+            smalllist.Add("Banshee", Banshee);
+            smalllist.Add("Demon", Demon);
+            smalllist.Add("Jinn", Jinn);
+            smalllist.Add("Mare", Mare);
+            smalllist.Add("Onie", Onie);
+            smalllist.Add("Phantom", Phantom);
+            smalllist.Add("Poltergeist", Poltergeist);
+            smalllist.Add("Revnant", Revenant);
+            smalllist.Add("Shade", Shade);
+            smalllist.Add("Spirit", Spirit);
+            smalllist.Add("Wraith", Wraith);
+            smalllist.Add("Yurei", Yurei);
+
+            ghostlist = smalllist;
+            return smalllist;
```

Comment:
You're both returning `smalllist`, as well as assigning it to `ghostlist`. Where `monsterlist()` is called on 27, you're not capturing the returned value. I suggest removing the return altogether